zcash_client_sqlite/src/wallet.rs

```diff
@@ -716,10 +716,8 @@ pub(crate) fn get_wallet_summary(
         let mut stmt_transparent_balances = conn.prepare(
             "SELECT u.received_by_account, SUM(u.value_zat)
              FROM utxos u
-             LEFT OUTER JOIN transactions tx
-             ON tx.id_tx = u.spent_in_tx
              WHERE u.height <= :max_height
-             AND tx.block IS NULL
+             AND u.spent_in_tx IS NULL
```

Comment:
I think the `expired unmined` could be handled by:
```suggestion
             LEFT OUTER JOIN transactions tx
             ON tx.id_tx = u.spent_in_tx
             WHERE u.height <= :max_height
             AND (u.spent_in_tx IS NULL OR (tx.block IS NULL AND tx.expiry_height < :max_height))
```

Reply:
There's maybe an off-by-one here (this might need to be `tx.expiry_height <= max_height`, but I'd have to test to confirm). But per our thread in Slack, this is insufficient and the full fix would need more wide-scale changes (complicated by the difference in data model between transparent and shielded).

Reply:
It actually needs to compare to the chain tip, not :max_height

Reply:
Changes from #988 have been brought into this PR.